profab/role/ami/ubuntu.py

```diff
@@ -16,7 +16,7 @@ class Configure(Role):
     def ami(self, region, bits, size):
         """Return the AMI that was passed in to the role.
         """
-        disk = 'ebs' if size == 't1.micro' else 'instance'
```

Comment:
Infra guy suggest me to do that, infra guy said that EBS can resize root volume but instance can't resize root volume.